### PR TITLE
Feature/set icomoon

### PR DESCRIPTION
**Pull request by turonchek**
- icons uploaded
- images uploaded

### CODE REVIEW

File: styles/_icomoon.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+.icon-chevron-left:before {
+  content: "\e903";
+}
+.icon-chevron-right:before {
+  content: "\e904";
+}
+.icon-fb:before {
+  content: "\e905";
+}
+.icon-footer_email:before {
```
Comment (Plakhotenko) [inline styles/_icomoon.scss]: just `icon-email`

File: styles/_icomoon.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  content: "\e906";
+}
+.icon-footer_phone:before {
```
Comment (Plakhotenko) [inline styles/_icomoon.scss]: `icon-phone`

File: styles/_icomoon.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  content: "\e907";
+}
+.icon-footer_pin:before {
```
Comment (Plakhotenko) [inline styles/_icomoon.scss]: `icon-pin`

File: styles/_icomoon.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+.icon-shopping-cart:before {
+  content: "\e90e";
+}
+.icon-star:before {
+  content: "\e90f";
+}
+.icon-user:before {
+  content: "\e910";
+}
+.icon-yt:before {
```
Comment (Plakhotenko) [inline styles/_icomoon.scss]: `yt`? what does it mean? Could you please rename this icon?

Comment (turonchek) [reply styles/_icomoon.scss]: It means youtube)) just short

**Review by Plakhotenko — APPROVED** [review]

**Pull request merged**